### PR TITLE
cli: adding prio fee support to validator-info

### PR DESCRIPTION
**Pull request by kaplanmaxe**
#### Problem

Adding priority fee support to `solana validator-info publish`

#### Summary of Changes

Adding priority fee support to `solana validator-info publish`

Fixes #

### CODE REVIEW

**Review by joncinque — CHANGES_REQUESTED** [review]
Thanks for your contribution! Can you run `./cargo nightly fmt --all` from the top of the repo to format? CI is failing due to the formatting issue right now

**Comment by kaplanmaxe** [comment]
Sorry about that @joncinque . Fixed, rebased, and squashed

File: cli/src/validator_info.rs [hunk 1]
```diff
@@ -3,6 +3,7 @@
 use {
     crate::{
         cli::{CliCommand, CliCommandInfo, CliConfig, CliError, ProcessResult},
+        compute_unit_price::WithComputeUnitPrice,
```
Comment (joncinque) [inline cli/src/validator_info.rs]: Crap, darn, I Just realized I broke this with #826, can you rebase and update this line?
```suggestion
        compute_budget::WithComputeUnitPrice,
```

**Review by joncinque — COMMENTED** [review]
The change looks good, but I realized it won't build at this point because of a change I just merged. Could you rebase one last time? Then I'll merge once it passes CI.

If that's too annoying, I can also take care of it

**Comment by kaplanmaxe** [comment]
No worries at all @joncinque . Just pushed. Thanks!

**Review by joncinque — APPROVED** [review]
Thanks again for doing the work!

**Pull request merged**